dos/src/main.rs

```diff
@@ -449,7 +449,9 @@ fn get_target(
                         Some((*node.pubkey(), node.tpu_forwards(protocol).unwrap()))
                     }
                     Mode::Repair => todo!("repair socket is not gossiped anymore!"),
-                    Mode::ServeRepair => Some((*node.pubkey(), node.serve_repair().unwrap())),
+                    Mode::ServeRepair => {
+                        Some((*node.pubkey(), node.serve_repair(Protocol::UDP).unwrap()))
+                    }
```

Comment:
i assume the plan is to next implement repair over quic?

Reply:
yes, that is the plan